packit/patches.py

```diff
@@ -505,41 +516,34 @@ def create_patches(
         :return: [PatchMetadata, ...] list of patches
         """
         files_to_ignore = files_to_ignore or []
+        patch_list: List[PatchMetadata] = []
         contained = self.are_child_commits_contained(git_ref)
+        patches_revision_range = f"{git_ref}..HEAD"
         if not contained:
-            self.linearize_history(git_ref)
+            lin_branch = self.linearize_history(git_ref)
+            # we're still on the same branch but want to get patches from the linearized branch
```

Comment:
:heart: this comment. Provides just the right context for something not that obvious.